configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
+
+The creation and validation of a certificate is outside the scope of this article and you must already have a signed certificate. 
+
+These instructions assume usage for secure network connections on the internet or corporate networks, but may also be used for other purposes like signing e-mail.   
+
+{{< altimg "network-overview.jpg" "help-center/configuration/configuring-network/" >}}
+
+WARNING: You must trust the source of any certificate to be installed. Using an insecure or otherwise compromised certificate is a critical security risk.
+
+## The NSS Database
+
+Solus utilizes Network Security Service(NSS) for managing digital certificates. Each system user account will have a user NSS database located at `$HOME/.pki/nssdb`.   
+
+The `certutil` utility is installed by default and can be verified with `which certutil`. We will be operating on the logged in users' NSSDB, without need for Root permissions. A summary of options can be shown with `--syntax`, and `-H` will show a complete list of option descriptions.
+
+### List Database Entities
+
+`$ certutil -d sql:$HOME/.pki/nssdb -L`
+
+#### Example Output
+
+This example shows an existing self-signed, trusted CA certificate as described by the SSL Trust Attribute `C`.
+
+```
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+
+mynew-Root-CA                                                C,,
+```
+
+### Add a Certificate
+
+`$ certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n <cert-name> -i <path_to_crt_file>`
+
+#### Example Output
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -A -t "C,," -n mynew-Root-CA -i mynew-Root-CA.CRT
+$ certuilt -d sql:$HOME/.pki/nssdb -L
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+
+mynew-Root-CA                                                C,,
+```
+
+### Validate the Certificate
+
+`$ certutil -d sql:$HOME/.pki/nssdb -V -u <certusage> -n <cert_name>`
+
+#### Example Output
+
+In this example, we specify the usage context with the `-u` option to "Verify CA" `Y`.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u Y
+certutil: certificate is valid
+```
+
+In this example, we try to validate the Root/CA certificate in the SSL client usage context.
+
+```
+$ certutil -d sql:$HOME/.pki/nssdb -V -n mynew-Root-CA -u C
+certutil: certificate is invalid: Certificate key usage inadequate for attempted operation.
+```
+
+### Using a Certificate
+
+Some client applications will need to be restarted to use the new CA certificate. Valid SSL or TLS host/service certificates including a signature with by the newly installed CA/Root certificate can now be validated and inspected. Web browsers provide utilities for certificate inspection, often with a status icon in the URL bar. 
```

Comment:
```suggestion
Some client applications will need to be restarted to use the new CA certificate. Valid SSL or TLS host/service certificates including a signature with the newly installed CA/Root certificate can now be validated and inspected. Web browsers provide utilities for certificate inspection, often with a status icon in the URL bar. 
```

Maybe also possible to lose the "with" instead? Or maybe restructure the sentence as a whole?

"certificates signed by the newly installed[...]"? Not sure if that encompasses everything you were trying to express here.